ruleset/rules/0095-sshd_rules.xml

```diff
@@ -480,17 +471,14 @@
     <description>sshd: connection reset</description>
   </rule>
 
-  <rule id="5763" level="5">
-    <if_sid>5750</if_sid>
-    <match>not listed in AllowUsers|listed in DenyUsers</match>
-    <description>sshd: Attempt to login using a denied user</description>
-    <group>invalid_login,pci_dss_10.2.4,pci_dss_10.2.5,gpg13_7.1,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
-  </rule>
-
-  <rule id="5764" level="10" frequency="8" timeframe="120" ignore="60">
-    <if_matched_sid>5763</if_matched_sid>
-    <description>sshd: Multiple access attempts using a denied user.</description>
-    <group>invalid_login,pci_dss_10.2.4,pci_dss_10.2.5,pci_dss_11.4,gpg13_7.1,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,nist_800_53_SI.4,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
+  <rule id="5763" level="10" frequency="8" timeframe="120" ignore="60">
+    <if_matched_sid>5760</if_matched_sid>
+    <description>sshd: brute force trying to get access to the system. Authentication failed.</description>
+    <same_source_ip />
+    <group>authentication_failures,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_SI.4,nist_800_53_AU.14,nist_800_53_AC.7,pci_dss_11.4,pci_dss_10.2.4,pci_dss_10.2.5,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
```

Comment:
Group label should go after MITRE label.

Reply:
done